src/clientMods.ts

```
@@ -0,0 +1,580 @@
+  }
+  if (mod.scriptMainUnstable) {
+    const blob = new Blob([mod.scriptMainUnstable], { type: 'text/javascript' })
+    const url = URL.createObjectURL(blob)
+    // eslint-disable-next-line no-useless-catch
+    try {
+      const module = await import(/* webpackIgnore: true */ url)
+      module.default?.(structuredClone(mod))
+      window.loadedMods[mod.name] ??= {}
+      window.loadedMods[mod.name].mainUnstableModule = module
+    } catch (e) {
+      throw e
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)